cmd/controller/root.go

```diff
@@ -320,6 +320,11 @@ var rootCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		if err := mgr.AddReadyzCheck("readyz", healthz.Ping); err != nil {
```

Comment:
Soooo... I'm confused. :D 

```go
		if err := mgr.AddHealthzCheck("healthz", healthz.Ping); err != nil {
			setupLog.Error(err, "unable to add controller healthz check")
			os.Exit(1)
		}

		if err := mgr.AddReadyzCheck("readyz", healthz.Ping); err != nil {
			setupLog.Error(err, "unable to add controller readyz check")
			os.Exit(1)
		}
```

This is my current view. These do the same thing, so I'm not sure what's the point of the readyz check?